packages/mdc-chips/README.md

```diff
@@ -40,25 +40,52 @@ npm install --save @material/chips
 
 ```html
 <div class="mdc-chip-set">
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
-  <div class="mdc-chip">
+  <div class="mdc-chip" tabindex="0">
     <div class="mdc-chip__text">Chip content</div>
   </div>
 </div>
 ```
 
+#### Leading and Trailing Icons
+
+You can optionally add a leading icon (i.e. thumbnail) and/or a trailing icon to a chip. To add an icon, add an `i` element with your preferred icon, give it a class of `mdc-chip__icon`, and a class of either `mdc-chip__icon--leading` or `mdc-chip__icon--trailing`. If you're adding a trailing icon, also set `tabindex="0"` and `role="button"` to make it accessible by keyboard and screenreader.
+
+##### Leading icon
+
+```html
+<div class="mdc-chip mdc-chip">
+  <i class="material-icons mdc-chip__icon mdc-chip__icon--leading">event</i>
+  <div class="mdc-chip__text">Add to calendar</div>
+</div>
+```
+
+##### Trailing icon
+
+```html
+<div class="mdc-chip mdc-chip">
+  <div class="mdc-chip__text">Jane Smith</div>
+  <i class="material-icons mdc-chip__icon mdc-chip__icon--trailing" tabindex="0" role="button">cancel</i>
+</div>
+```
+
 ### CSS Classes
 
 CSS Class | Description
 --- | ---
+`mdc-chip-set` | Mandatory. Indicates the set that the chip belongs to
```

Comment:
Can we change `.mdc-chip-set` to use `display: flex; flex-wrap: wrap` to collapse the whitespace in the markup?

It will give us much better control over spacing, so we can match the spec exactly.

Reply:
I put it in #2265 since it requires the creation of a new file (mdc-chip-set/mdc-chip-set.scss)